### DISCUSSION TITLE
Dynamic default workspaces, and creating workspaces on the fly

### DISCUSSION THREAD

**Discussion by Shifta-Robel** [nvim-neorg/neorg#1032] [Ideas]
I recently started using Neorg, I want use it more to organize notes and tasks by directory, but here's what's making that hard
 - I have to manually select the workspace each time, rather than having it default to the workspace in my **current directory or closest parent directory**, It would be great if the default workspace could be set dynamically based on my current location.
 - The only way I've found to create new workspaces is through my nvim configs, this feels like a lot of ceremony. I'd love an easier way to generate a workspace directly from Neorg in my current directory without needing to update my configs. 

 Or do these go against the neorg philosopy?
 I'm new to Neorg so any suggestions would be much appreciated!

**Comment by pysan3** [comment]
Fixed with this PR. Use `:Neorg index` to open index.norg inside the current directory. It doesn't have fancy parent tree lookup tho. 

https://github.com/nvim-neorg/neorg/pull/1131

As I said in the PR, automatically creating index file is disabled to avoid creating index files everywhere by mistake. You'll have to create index.norg manually beforehand to work with it.
